android/src/main/java/com/github/yamill/orientation/listeners/OrientationConfigListener.kt

```diff
@@ -41,6 +39,9 @@ class OrientationConfigListener internal constructor(
         val activity = onGetCurrentActivity()
         if (activity != null) {
             activity.registerReceiver(receiver, IntentFilter(INTENT_ACTION_CONFIG_CHANGED))
+            val orientationInt = reactContext.resources.configuration.orientation
```

Comment:
nit: Does 

```
val orientationInt = reactContext.resources.configuration.orientation
            val orientation = OrientationUtil.getOrientationString(orientationInt)
            tryEmitOrientationChange(orientation, reactContext)
```

need to be inside the `if (activity != null) {` block? It doesn't seem to use the `activity`, so the activity-null-check may not be necessary for emitting the orientation change

Reply:
Hmm yes, that's definitely true.  I think some of my wires got cross when migrating my diffs from the lib code to this repo.